src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Input/Commands/GenerateNuGetPackageProjectContextMenuCommand.cs

```diff
@@ -7,7 +7,7 @@
 namespace Microsoft.VisualStudio.ProjectSystem.VS.Input.Commands
 {
     [ProjectCommand(ManagedProjectSystemPackage.ManagedProjectSystemCommandSet, ManagedProjectSystemPackage.GenerateNuGetPackageProjectContextMenuCmdId)]
-    //[AppliesTo(ProjectCapability.GenerateNuGetPackage)]
+    //[AppliesTo(ProjectCapability.Pack)]
```

Comment:
Yes - https://github.com/dotnet/roslyn-project-system/issues/998. Will add it to the comment